src/main/webapp/app/exercises/modeling/assess/modeling-assessment.component.ts

```diff
@@ -182,7 +189,7 @@ export class ModelingAssessmentComponent implements AfterViewInit, OnDestroy, On
                 }
                 feedback.credits = assessment.score;
                 feedback.text = assessment.feedback;
-                if (assessment.dropInfo && assessment.dropInfo.instruction.id) {
+                if (assessment.dropInfo && assessment.dropInfo.instruction?.id) {
```

Comment:
why to we need the `?`, the interface DropInfo is not defined in a way that instruction could be undefined